recipes/tensorflow-lite/all/conanfile.py

```diff
@@ -76,60 +65,69 @@ def configure(self):
+            "TFLITE_ENABLE_NNAPI": self.options.get_safe("with_nnapi", False),
+            "TFLITE_ENABLE_GPU": False,
+            "TFLITE_ENABLE_XNNPACK": self.options.with_xnnpack,
+            "TFLITE_ENABLE_MMAP": self.options.get_safe("with_mmap", False),
+            "FETCHCONTENT_FULLY_DISCONNECTED": True,
+            "clog_POPULATED": True,
+        })
+        if self.settings.arch == "armv8":
+            # Not defined by Conan for Apple Silicon. See https://github.com/conan-io/conan/pull/8026
+            tc.variables["CMAKE_SYSTEM_PROCESSOR"] = "arm64"
+        tc.generate()
+        deps = CMakeDeps(self)
+        deps.generate()
+
     def validate(self):
         if self.settings.compiler.get_safe("cppstd"):
-            tools.check_min_cppstd(self, 17 if Version(self.version) >= "2.9.1" else 14)
+            check_min_cppstd(self, 17)
 
         minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
         if not minimum_version:
-            self.output.warn(f"{self.name} requires C++14. Your compiler is unknown. Assuming it supports C++14.")
+            self.output.warn(f"{self.name} requires C++17. Your compiler is unknown. Assuming it supports C++17.")
         elif Version(self.settings.compiler.version) < minimum_version:
-            raise ConanInvalidConfiguration(f"{self.name} requires C++14, which your compiler does not support.")
+            raise ConanInvalidConfiguration(f"{self.name} requires C++17, which your compiler does not support.")
 
     def source(self):
-        files.get(self, **self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
 
     def build(self):
-        files.apply_conandata_patches(self)
-        cmake = self._configure_cmake()
-        cmake.build()
-
-    @functools.lru_cache(1)
-    def _configure_cmake(self):
+        apply_conandata_patches(self)
         cmake = CMake(self)
-        cmake.definitions.update({
-            "CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS": True,
-            "TFLITE_ENABLE_RUY": self.options.with_ruy,
-            "TFLITE_ENABLE_NNAPI": self.options.get_safe("with_nnapi", False),
-            "TFLITE_ENABLE_GPU": False,
-            "TFLITE_ENABLE_XNNPACK": self.options.with_xnnpack,
-            "TFLITE_ENABLE_MMAP": self.options.get_safe("with_mmap", False)
-        })
-        if self.settings.arch == "armv8":
-            # Not defined by Conan for Apple Silicon. See https://github.com/conan-io/conan/pull/8026
-            cmake.definitions["CMAKE_SYSTEM_PROCESSOR"] = "arm64"
-        cmake.configure(build_folder=self._build_subfolder)
-        return cmake
+        cmake.configure(build_script_folder=join("tensorflow", "lite"))
+        cmake.build()
 
     @staticmethod
-    def _create_cmake_module_alias_target(module_file):
+    def _create_cmake_module_alias_target(self, module_file):
```

Comment:
```suggestion
    def _create_cmake_module_alias_target(conanfile, module_file):
```
Avoid shadowing names with static methods